src/types.rs

```diff
@@ -258,6 +267,8 @@ pub struct FullCrate {
     pub downloads: Downloads,
     pub owners: Vec<User>,
     pub reverse_dependencies: Vec<Dependency>,
+    pub reverse_dependencies_versions: Vec<Version>,
+    pub reverse_dependencies_meta: Meta,
```

Comment:
that field is not really needed since we always fetch all dependencies anyway, so the `total` information would just by a `.len()` away.